src/utils.js

```diff
@@ -99,6 +99,49 @@ const logFilenameFlag = Flags.string({
 	required: true,
 });
 
+// The `destinations` object to hold the configuration for log forwarding destinations.
+// It prompts for the required inputs for the destination.
+// Each destination can have different key/value pairs of configuration credentials.
+// and applies the validation logic accordingly.
+const destinations = {
+	// Configuration for the 'newrelic' destination
+	newrelic: {
```

Comment:
I'd prefer this is to be "New Relic" although we are internally processing it as "newrelic"